Gemfile

```diff
@@ -36,6 +35,8 @@ gem 'passenger'
 # MySQL
 gem 'mysql2'
 
+gem 'linkeddata', '~> 1.1.11.1', :github => 'ruby-rdf/linkeddata'
```

Comment:
Apparently there was a dependency on the version 1.1.11.1, Circle kept on complaining that it couldn't download this version from rubygems, so in order to make circle pass I had to explicitly grab this version from github. Is there an alternate way I could accomplish this?